Formula/flarectl.rb

```diff
@@ -0,0 +1,28 @@
+class Flarectl < Formula
+  desc "CLI application for interacting with a Cloudflare account"
+  homepage "https://github.com/cloudflare/cloudflare-go/tree/master/cmd/flarectl"
+  url "https://github.com/cloudflare/cloudflare-go/archive/v0.11.6.tar.gz"
+  sha256 "def3f0b64da5a473a942303d297ccab072044856ca023fe60a435e4145aadcfd"
+  head "https://github.com/cloudflare/cloudflare-go.git"
+
+  depends_on "go" => :build
+  depends_on "gox" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/cloudflare/cloudflare-go"
+    dir.install buildpath.children
+
+    cd dir do
+      system "gox", "-arch", "amd64", "-os", "darwin", "-ldflags", "\"-X main.Rev=#{version}\"", "./..."
```

Comment:
Since this will run on macOS does it even need gox?

Reply:
gox is the build system that upstream uses, I haven't had success building this otherwise, but it's been a while.

Reply:
Removed gox dependency